### DISCUSSION TITLE
Apache Support

### DISCUSSION THREAD

**Discussion by ddshore** [openappsec/openappsec#230] [Feature Requests]
Hi, 

Are you guys working on support for Apache http server?  I understand your current focus is in nginx and supporting nvoy in the future, however, I do not see any indication that apache might be down the road map.

**Comment by bilbogh** [comment]
Currently it's not on the roadmap, but we will be glad to support anyone from the community that would like to work on it.

**Comment by orianelou** [comment]
If you're interested in this feature, please 👍 or upvote this issue to show your support!
